cloudify_aws/codepipeline/resources/pipeline.py

```diff
@@ -72,6 +74,20 @@ def delete(self, params=None):
                                             params=params))
         self.client.delete_pipeline(**params)
 
+    def execute(self, name=None, clientRequestToken=None):
+        """
+            start execution of an existing Pipeline.
+        """
+        params = {'name': name if name else self.resource_id}
+        if clientRequestToken:
+            params.update({"clientRequestToken": clientRequestToken})
```

Comment:
Is this a confidential asset?

Reply:
I didn't fully understood what this token used for and if it confidential. 
https://boto3.amazonaws.com/v1/documentation/api/1.12.13/reference/services/codepipeline.html#CodePipeline.Client.start_pipeline_execution
https://docs.aws.amazon.com/codepipeline/latest/APIReference/API_StartPipelineExecution.html

what do you think?